x-pack/plugins/apm/public/components/app/ServiceOverview/view.tsx

```diff
@@ -67,7 +47,11 @@ export class ServiceOverview extends Component<Props, State> {
           render={() => (
             <ServiceList
               items={this.props.serviceList.data}
-              noItemsMessage={noItemsMessage}
+              noItemsMessage={
+                <NoServicesMessage
+                  hasDataEver={this.state.historicalDataFound}
```

Comment:
I don't have any prefs for `hasDataEver` or `historicalDataFound` but let's pick one to keep consistent terminology

Reply:
... and let's make it clear that it's _agent_ data, and not just any data.

Reply:
I'm just putting it back to `historicalDataFound` for now, I don't think this is very important.

Reply:
This part continues to be a source of confusion, and I think good naming and/or comments can help a lot. If we stick to `historicalDataFound` we at least don't muddy the waters even more.